WDL.YAML-tmLanguage

```diff
@@ -75,7 +77,7 @@ repository:
   keywords:
     patterns:
     - name: keyword.other.wdl
-      match: \s*(call|command|output|runtime|task|workflow|if|then|else|import|as|input|output|meta|parameter_meta|scatter)\s+
```

Comment:
I see `command` defined below, but is it also necessary to remove `output` here?

Reply:
It was in here twice

Reply:
🤦‍♂️